src/api/modelTypes.js

```diff
@@ -242,6 +242,7 @@ export type Stream = {|
   invite_only: boolean,
   is_announcement_only: boolean,
```

Comment:
This should become optional now, with a comment saying when it was removed.

Reply:
I checked on `2.2-dev-2006-g0cf4c89713`, we are still getting `is_announcement_only `. 
My main concern is, should we make it optional as part of this change because it will affect `getIsActiveStreamAnnouncementOnly` which is used in `ComposeBox.`

Reply:
I see. OK, if the server is still sending it then it wasn't removed and we don't need to mark it optional. Someday (zulip/zulip#14188) there'll be API docs for this.